migcontrol/templates/base.html

```diff
@@ -25,7 +25,15 @@
   {% endcompress %}
 
 </head>
+<style>
 
+#form{
+	
+	margin-left:32%;
+
+}
+
+</style>
```

Comment:
It's nice that you have thought to make an exceptional rule to this, but it's not exactly what we are looking for.

Firstly, the design has to be responsive. So for instance on mobile, you can't have a fixed left margin.

Secondly, having an inline `<style>` tag with rules that only apply to one page isn't the way to go. But have a look in the .scss files: https://github.com/migcontrol/django-migcontrol/tree/main/migcontrol/static/scss

In `main.scss`, you find the entrypoint for loading all the SCSS, which is then pre-compiled into CSS (by the django-compressor dependency)

![image](https://user-images.githubusercontent.com/374612/168031148-8dba69fe-c928-457b-85da-4a870aeb9c75.png)

I don't think that we need a special rule for this (I can explain below), but *if* we did, we should add it to the right file here or create a new file with a good name.